load.php

```diff
@@ -137,6 +137,41 @@ function( $module_settings ) {
 	return $modules;
 }
 
+/**
+ * Gets the active and valid performance modules.
+ *
+ * @since n.e.x.t
+ *
+ * @return array List of active valid module slugs.
+ */
+function perflab_get_active_and_valid_modules() {
+	$active_modules = perflab_get_active_modules();
```

Comment:
Suggestion accepting an array as parameters here removes the need that this function those 2 things get active modules and filter to valid ones instead my suggestion would be to accept an array of modules and focus this function on only returning the active ones based on the provided parameter, for instance:


```suggestion
function perflab_get_valid_modules( array $modules = array() ) {
```

That way this function is decoupled from the need to be linked to the `active` modules and you can pass any value in your test to verify the behavior is independent from the active modules.

Reply:
Thanks for the feedback.

Yes, this change will help with the unit tests. The PR was updated with the changes.